client/ionic/src/App.tsx

```diff
@@ -32,8 +35,15 @@ const App: React.FC = () => (
       <IonRouterOutlet>
         <Route path="/about" component={About} exact={true} />
         <Route path="/menu" component={Menu} exact={true} />
+        <Route
+          path="/protect_yourself"
```

Comment:
Just a note that in #147 we're using hyphenated URL paths like `/coping-parents` so we should probably be consistent in whichever one we choose.